apps/admin/frontend/src/screens/unconfigured_screen.tsx

```diff
@@ -71,64 +54,56 @@ export function UnconfiguredScreen(): JSX.Element {
     []
   );
   const [isLoading, setIsLoading] = useState(false);
-  const [vxElectionFileIsInvalid, setVxElectionFileIsInvalid] = useState(false);
-  const [systemSettingsFileIsInvalid, setSystemSettingsFileIsInvalid] =
-    useState(false);
+  const [configureError, setConfigureError] = useState<ConfigureError>();
   const [isUsingConverter, setIsUsingConverter] = useState(false);
 
   const client = useMemo(
     () => getElectionDefinitionConverterClient(converter),
     [converter]
   );
 
-  const saveSystemSettingsToBackend = useCallback(
-    (fileContent: string) => {
-      setSystemSettingsMutation.mutate(
-        { systemSettings: fileContent },
-        {
-          onSuccess: (result) => {
-            if (result.isErr()) {
-              setSystemSettingsFileIsInvalid(true);
-            }
-          },
+  const configureMutateAsync = configureMutation.mutateAsync;
+  const configureFromElectionPackage = useCallback(
+    async (
+      electionData: string,
+      systemSettingsData: string = JSON.stringify(DEFAULT_SYSTEM_SETTINGS)
+    ) => {
+      setConfigureError(undefined);
+      try {
+        const result = await configureMutateAsync({
+          electionData,
+          systemSettingsData,
+        });
+        if (result?.isErr()) {
+          setConfigureError(result.err());
+          // eslint-disable-next-line no-console
+          console.error(
+            'configure failed in saveElectionToBackend',
+            result.err().message
+          );
         }
-      );
+        return result;
+      } catch {
+        // Do nothing
```

Comment:
Since the default React query error handling will kick in, the "Handled by default query client error handling" comment that we typically use might be more informative. While it's true that we're doing nothing in the catch, the current comment might be misleading as the failures here won't actually be silent

Reply:
Thanks, will update!